lib/pantry.rb

```diff
@@ -0,0 +1,22 @@
+class Pantry
+    attr_reader :stock
+
+    def initialize
+        @stock = Hash.new(0)   
+    end
+
+    def stock_check(ingredient)
+        @stock[ingredient]
+    end
+    
+    def restock(ingredient, quantity)
+        @stock[ingredient] += quantity
+    end
+
+    def enough_ingredients_for?(recipe)
+        if self.stock.include?(recipe.ingredients_required)
```

Comment:
We don't want to just check if the stock includes it, we also need to check that the quantity in stock is more than or equal to the amount needed.